app/controllers/products_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProductsController < ApplicationController
+  def edit
+    @product = Product.find(params[:id])
+  end
+
+  def update
+    @product = Product.find(params[:id])
+    @product.update(product_update_params[:product])
+
```

Comment:
Remove random white space. This is nit-picky but I think in internships your teams will care about this so it's good to break the habit now. I see it in a few other places, but Riley also commented something similar, so you get the idea. 🌞